library/src/main/java/com/airbnb/android/airmapview/AirMapView.java

```diff
@@ -80,6 +80,18 @@ public void initialize(FragmentManager fragmentManager, AirMapInterface mapInter
    * @param fragmentManager required for initialization
    */
   public void initialize(FragmentManager fragmentManager) {
+    initialize(fragmentManager, null, null);
+  }
+
+  /**
+   * Used for initialization of the underlying map provider. Uses the default preferred map provider
+   * (currently Native Google Maps, then Mapbox Web maps, then Web Google Maps).
+   *
+   * @param fragmentManager required for initialization
+   * @param mapboxAccessToken Mapbox Access Token (required for Mapbox maps)
+   * @param mapboxMapId Mapbox Map Id (required for Mapbox maps)
+   */
+  public void initialize(FragmentManager fragmentManager, String mapboxAccessToken, String mapboxMapId) {
```

Comment:
I'm happy to move this to `MapboxMapViewBuilder`, but could you suggest how / where you'd like to see that be instantiated?

As of right now according the Sample App's [`MainActivity onCreate()'](https://github.com/airbnb/AirMapView/blob/cc055292909e598dd4c74b6d65fad67e6f5179c9/sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java#L39-L55) there's 2 ways into the setup process (ie what creates `WebAirMapViewBuilder` or potentially a `MapboxMapViewBuilder`.)  

1. [DefaultAirMapViewBuilder](https://github.com/airbnb/AirMapView/blob/cc055292909e598dd4c74b6d65fad67e6f5179c9/sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java#L43)
1. [AirMap.initialiaze()](https://github.com/airbnb/AirMapView/blob/cc055292909e598dd4c74b6d65fad67e6f5179c9/sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java#L54)

Neither of these ways allows for external credentials to be passed in to the library so that the runtime determination on Google Plays Services can work (ie use Google Native if available, if not use Mapbox Web).  The only way around it would be to explicitly set an AirMapInterface via [AirMapView.initialize(FragmentManager fragmentManager, AirMapInterface mapInterface)](https://github.com/airbnb/AirMapView/blob/cc055292909e598dd4c74b6d65fad67e6f5179c9/library/src/main/java/com/airbnb/android/airmapview/AirMapView.java#L61-L74) which would then immediately load the Mapbox Web.

Thoughts?